shell/browser/feature_list.cc

```diff
@@ -43,9 +43,6 @@ void InitializeFeatureList() {
 
 #if BUILDFLAG(IS_WIN)
   disable_features +=
-      // Disable async spellchecker suggestions for Windows, which causes
-      // an empty suggestions list to be returned
-      std::string(",") + spellcheck::kWinRetrieveSuggestionsOnlyOnDemand.name +
```

Comment:
@VerteDinde @MarshallOfSound given the feature flag is no longer available, I have attempted an alternative approach similar to what chrome does in https://github.com/electron/electron/pull/45055/commits/47f08d8922aa909f62ecf0fca75179f160d06876, please check if that helps.